src/ks-app/package.json

```diff
@@ -21,10 +21,10 @@
     "@angular/platform-browser": "^5.0.0",
     "@angular/platform-browser-dynamic": "^5.0.0",
     "@angular/router": "^5.0.0",
-    "@webcomponents/custom-elements": "1.0.0",
     "@clr/angular": "file:../../dist/clr-angular/clr-angular-0.11.7-patch.1.tgz",
     "@clr/icons": "file:../../dist/clr-icons",
     "@clr/ui": "file:../../dist/clr-ui",
+    "@webcomponents/custom-elements": "1.0.0",
```

Comment:
this change is not related to this PR. if we revert the main fix, then this will revert too. this is not a major issue here but just something to note for the future.

Reply:
Sorry, I missed it when I split out the changes for the other PR that I'm working on.